src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -14,29 +17,78 @@
 @CrossOrigin(origins = "http://localhost:3000")
 public class EnrollmentController {
 
+    @Autowired
+    EnrollmentRepository enrollmentRepository;
+
     // instructor downloads student enrollments for a section, ordered by student name
     // user must be instructor for the section
     @GetMapping("/sections/{sectionNo}/enrollments")
     public List<EnrollmentDTO> getEnrollments(
```

Comment:
Missing `sectionNo` check.
Postman link : `localhost:8080/sections/8222/enrollments`
Postman Output:
```json
[]
```
Expected: 404 or bad request

Reply:
Good catch @SMCaseyCode. The question is what we want to consider as an error here. We could handle this on the front end by checking the length of the array, or just display an empty list of enrollments. Team, what do you think? Should we throw an error here when no enrollments are found?